### ISSUE TITLE
Add Option to Filter Manga by Artist

### ISSUE THREAD

**Issue by BoneMarrow10** [KotatsuApp/Kotatsu#878]
### Describe your suggested feature

Please add the ability to search or filter manga base on the artist name.

![Screenshot_20240513-012638~2](https://github.com/KotatsuApp/Kotatsu/assets/156551596/42d0b20c-84a7-4582-bb45-34731739795f)
![Screenshot_20240513-012659~2](https://github.com/KotatsuApp/Kotatsu/assets/156551596/876db2ad-f6a9-4c32-9c83-f71606a63b32)

It's kinda a hassle when I want to find other manga they made that I have to close the app and jump into web browser.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by NagaYZ** [comment]
More of a parser issue, and only possible if the source supports this kind of search.

**Comment by Koitharu** [comment]
Yeah, this is an parsers-specific functionality, not sure it is worth to do it because only small part of manga sources will support it
